pycaching/geocaching.py

```diff
@@ -356,6 +363,51 @@ def search_quick(self, area, *, strict=False, zoom=None):
 
     # add some shortcuts ------------------------------------------------------
 
+    def search_rect(self, rect: Rectangle, *, per_query: int = 50, sortby: str = 'datelastvisited', origin=None):
+        """
+        Return a generator of caches in given Rectange area.
+
+        :param rect: Search area.
+        :param per_query: Number of caches requested in single query.
+        :param sortby: Order cached by given criterion.
```

Comment:
Please use a consistent naming style. This should probably be `sort_by` instead.

Reply:
fixed